docs/guides/migration-guide.md

```diff
@@ -117,6 +117,31 @@ Implementing the new session key protocol changes:
 - Plan expiration times based on your operational needs
 - Application-scoped keys track cumulative spending against allowances
 
+#### Auth Verify Changes
+
+The `createEIP712AuthMessageSigner` function signature has changed to align with the new session key structure.
+
+```typescript
+const eip712SigningFunction = createEIP712AuthMessageSigner(
+    walletClient,
+    {
+        scope: authMessage.scope,
+        // remove-next-line
+        application: authMessage.application,
+        // remove-next-line
+        participant: authMessage.session_key,
+        // remove-next-line
+        expire: authMessage.expire,
+        // add-next-line
+        session_key: authMessage.session_key,
+        // add-next-line
+        expires_at: authMessage.expires_at,
+        allowances: authMessage.allowances,
```

Comment:
Maybe it is better to use the "git diff" style for "before"/"after" comparison, so that "before" is red (akin to "removed") and "after" is green (akin to "added")?

Reply:
I wanted it to look more like git diff which is more granular for the changes
And this reduces the cognitive load on the viewer to look between two blocks and compare which parameter is not needed and which stays as it is.
I have edited the code snippets as per that rationale